zuul.d/jobs.yaml

```diff
@@ -340,3 +340,13 @@
     nodeset: ubuntu-xenial-otc
     secrets:
       - opentelekomcloud_credentials
+
+- job:
+    name: bosh-openstack-cpi-release-acceptance-test
```

Comment:
please add openstack version `newton` in job name, like: bosh-openstack-cpi-release-acceptance-test-newton